packages/gasket-plugin-nextjs/generator/redux/redux/store.js

```diff
@@ -0,0 +1,17 @@
+const { configureMakeStore } = require('@gasket/redux');
+const { HYDRATE, createWrapper } = require('next-redux-wrapper');
+{{{reduxReducers.imports}}}
+
+// Basic hydrate reducer for next-redux-wrapper
+// @see: https://github.com/kirill-konshin/next-redux-wrapper#usage
+const rootReducer = (state, { type, payload }) => type === HYDRATE ? { ...state, ...payload } : state;
```

Comment:
As a bit of information. I had started down the path of hiding this next-redux-wrapper boilerplate along these lines: https://github.com/godaddy/gasket/compare/redux-refresh...redux-refresh-alt

But it felt dirty. Yes, it makes the store a little nicer to read, but this is the only pro. By generating, the pros are it removes assumptions on future next-redux-wrapper changes, allows apps to upgrade/downgrade next-redux-wrapper at their leisure, and provides a starting point for tuning the HYDRATE reducer as needed.

I know we're trying to avoid boilerplate with Gasket, but this feels like the best approach for this situation to me. Open to discussion, however, if there are strong opinions otherwise.

Reply:
I think that works